proposals/NNNN-SendLocation-for-Mobile-Nav.md

```diff
@@ -0,0 +1,126 @@
+ applications that the system's capability has changed by sending the following new notification:
+  ```xml
+  <function name="OnSystemCapabilityChange" functionID="OnSystemCapabilityChangeID" messagetype="notification">
+     <description>Notification from the module when the module's capability has changed.</description>
+     <param name="systemCapability" type="SystemCapability" mandatory="false" array="true">
+     </param>
+ </function>
+  ```
+  i) SDL Core shall send this notification to all registered applications.  
+  ii) SDL Core shall send this notification only when the Service Application becomes active. In the case of SendLocation for Mobile Nav, the app can service the request only when the Service Application (Mobile Nav) app is active (i.e. in HMI state of FULL/LIMITED).  
+  iii) SDL Core shall only send this notificaiton when there is a change in system's capability. For example, if the system has an embedded navigation source which supports SendLocation and then the user activates a Mobile Nav app which also supports SendLocation then there is no actual change in the capability of the system and hence SDL Core shall not send the notification in this case.
+  
+2) *There must be a mechanism for the request/response from one app to be routed to another app.*  
+a)The proposed solution is to re-use the existing `SendLocation` RPC and have the ability for SDL Core to send this RPC as a request to the Service Application and for the Service Application to provide the `SendLocation` response. SDL Core will then need to provide the response provided by the Service Application as the response to the original requesting application.  
+b)SDL Core will need to track the request coming from an application to the request it sends to the Service Application and the response coming from the Service Application to the response that needs to be sent to the original requesting application.  
+c)When SDL Core re-routes the request to the Service Application, it should add a valid Correlation ID unlike the '-1' value it sends for notifications. The suggestion is for SDL Core to maintain its own Correlation ID which will be incremented by 1 whenever SDL Core will need to send a request out to a Service Application. The Service Application should provide the same Correlation ID in its response.  
+d)SDL Core should start a timer waiting for the response from the Service Application. If the timer expires before the SDL Core receives a response from the Service Application, then SDL Core should provide a response of `TIMED_OUT` to the requesting application.  
+e)The value of this timer should be carefully selected to cover various scenarios.  
+f)If the Service Application is not in FULL (i.e. being actively viewed on HMI) and this application receives a SendLocation request, the Service Application can draw attention to the user by using an Alert RPC informing that there is a destination informaiton ready to be viewed on the app.
```

Comment:
Please update to "...informing that there is destination information ready to be viewed on the app."